nessai/plot.py

```diff
@@ -498,7 +498,10 @@ def plot_trace(
     axes[-1].invert_xaxis()
 
     if filename is not None:
-        fig.savefig(filename, bbox_inches="tight")
+        try:
+            fig.savefig(filename, bbox_inches="tight")
+        except ValueError:
+            pass
```

Comment:
I find silent "failures" like this can be quite confusing, could you add a logging statement so the user knows it has failed? Maybe something like:

```python
logger.warning("Could not save the trace plot")
```
or more verbose if you want.